spec/src/main/asciidoc/metrics.asciidoc

```diff
@@ -140,7 +140,7 @@ a| * `method` - the fully qualified method name
 | Name | `ft.circuitbreaker.state.total`
 
 | Type in MP Metrics | `Gauge<Long>`
-| Type in MP Telemetry | `ObservableLongGauge`
+| Type in MP Telemetry | `A counter that emits long`
```

Comment:
Should there be a corresponding TCK change for this?

I'd have expected changes in `TelemetryMetricGetter` and `TelemetryMetricDefinition` to reflect that a counter is now expected.

`TelemetryMetricDefinition` also contains the `metricClass` field and we're now not specifying the exact class, though the class is only used to determine whether the metric is a counter, a gauge or something else.